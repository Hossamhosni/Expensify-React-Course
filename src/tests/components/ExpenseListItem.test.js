import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseListItem', () => {
	const expense = {
		description: 'Test desc.',
		amount: 22222,
		createdAt: 333,
		id: '10'
	};
	const wrapper = shallow(<ExpenseListItem {...expense}/>);
	expect(wrapper).toMatchSnapshot();
});