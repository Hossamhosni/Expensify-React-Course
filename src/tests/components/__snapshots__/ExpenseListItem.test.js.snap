// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/10"
  >
    <h3>
      Test desc.
    </h3>
  </Link>
  <p>
    22222
     - 
    333
  </p>
</div>
`;
